src/messages/email/content.ts

```diff
@@ -1,13 +1,22 @@
+import jwt from "jsonwebtoken"
+
+const decodedPassword = (password) => {
+  const { NODEMAILER_SECRET } = process.env
+  return jwt.verify(password, NODEMAILER_SECRET)
+}
+
 export const inviteEmailContent = {
   from: "AceleraMais <aceleradorametodosageis@gmail.com>",
   subject: "AceleraMais: Convite para mentora avaliadora!",
-  content: (name, password) => {
+  content: (user) => {
+    const { name, password } = user
+
     return `
     Olá ${name},
     Você foi convidada para ser mentora avaliadora no Acelera Mais. Uhuuuu!!!!
     Por favor, acesse https://aceleramais.com.br, faça o login com o seu e-mail usando a senha abaixo.
     Você precisará trocar a senha no primeiro login.
-    Senha gerada: ${password}
+    Senha gerada: ${decodedPassword(password)}
```

Comment:
Faz parte do conteudo saber o que ele precisa enviar, então achei mais interessante passar apenas o usuário e extrair dele o que for necessário para conter na mensagem. E deixei a função content para administrar a decodificação das senhas que virão pelo `user`.

Isso simplifica um pouco o metodo `sendEmail` na `userService`